QUEST_LV_0100.tsv

```diff
@@ -7906,146 +7906,146 @@ QUEST_LV_0100_20150804_007901	$Historian Rexipher is thinking deeply about somet
+QUEST_LV_0100_20150908_007980	$You are enlightened after checking what was written on the tombstones of Lydia Schaffen{nl}You receive 1 status point!
+QUEST_LV_0100_20150908_007981	$Preparing a live sacrifice
+QUEST_LV_0100_20150908_007982	$Talking
+QUEST_LV_0100_20150908_007983	$Stop the Guardians from destroying the Royal Mausoleum!
+QUEST_LV_0100_20150908_007984	$It seems that the Royal Mausoleum will collapse soon!{nl}Stop the Guardians attacking it!
+QUEST_LV_0100_20150908_007985	$You can hear someone crying out!
+QUEST_LV_0100_20150908_007986	$Placing the Soul Jar
+QUEST_LV_0100_20150908_007987	$A portal to the entrance to the Royal Mausoleum has been opened!
+QUEST_LV_0100_20150908_007988	$Charging Power
+QUEST_LV_0100_20150908_007989	$Handing over the activating stones
+QUEST_LV_0100_20150908_007990	$The Guardian of the Royal Family Treasure opened its mouth
+QUEST_LV_0100_20150908_007991	$Destroy the corrupted stone lantern!
+QUEST_LV_0100_20150908_007992	$This isn't the only stone lantern{nl}Find the other corrupted stone lanterns and destroy them
+QUEST_LV_0100_20150908_007993	$Obtaining the marks' power
+QUEST_LV_0100_20150908_007994	$The powerful aura of Wheelen is surrounding your body
+QUEST_LV_0100_20150908_007995	$Defeat the Guardian of the Royal Mausoleum who is attacking!
+QUEST_LV_0100_20150908_007996	$It's a trap! Defeat the monsters that are rushing in!
+QUEST_LV_0100_20150908_007997	$You didn't find anything!
+QUEST_LV_0100_20150908_007998	$As you activated the device, the portal has opened!{nl}Check the location of the portal from the map!
+QUEST_LV_0100_20150908_007999	$Follow Rexipher and go to the 3rd floor of the Royal Mausoleum!
 QUEST_LV_0100_20150908_008000	{memo X}
-QUEST_LV_0100_20150908_008001	You should reach the altar of Gedula before Rexipher!
-QUEST_LV_0100_20150908_008002	Activating the device
-QUEST_LV_0100_20150908_008003	Removing the restraining sphere
-QUEST_LV_0100_20150908_008004	Use the portal to move to Gate of the Great King
-QUEST_LV_0100_20150908_008005	Use the cable car to move to across Zvelgian Vacant Lot
-QUEST_LV_0100_20150908_008006	Go to the Holy Pond for the purification
-QUEST_LV_0100_20150908_008007	Purifying
-QUEST_LV_0100_20150908_008008	The portal is not working
-QUEST_LV_0100_20150908_008009	Find the elderly of the village in Vieta Gorge!
+QUEST_LV_0100_20150908_008001	$You should reach Gedulah Altar before Rexipher!
+QUEST_LV_0100_20150908_008002	$Activating the Device
+QUEST_LV_0100_20150908_008003	$Eliminating the Restraining Circle
+QUEST_LV_0100_20150908_008004	$Use the portal to move to Gate of the Great King
+QUEST_LV_0100_20150908_008005	$Use the cable car to move to across Zvelgian Vacant Lot
+QUEST_LV_0100_20150908_008006	$Go purify the Holy Pond
+QUEST_LV_0100_20150908_008007	$Purifying
+QUEST_LV_0100_20150908_008008	$The portal is not working
+QUEST_LV_0100_20150908_008009	$Go to meet the elder of the village at Vieta Gorge!
```

Comment:
Go to meet the elder of the village at Vieta Gorge! -> Find the elder of the village at Vieta Gorge!